Source/Core/Core/HW/WiimoteEmu/EmuSubroutines.cpp

```diff
@@ -620,4 +478,4 @@ void Wiimote::RealState()
     g_wiimotes[m_index]->EnableDataReporting(m_reporting_mode);
```

Comment:
has this code (savestate restore on real wiimote) been verified to still be adequate?

Reply:
Real wiimote savestate restore has never been adequate. 😛